modules/swagger-codegen/src/main/resources/android-volley/build.mustache

```diff
@@ -0,0 +1,91 @@
+{{#useAndroidMavenGradlePlugin}}
+group = '{{groupId}}'
+project.version = '{{artifactVersion}}'
+{{/useAndroidMavenGradlePlugin}}
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:1.2.2'
+        {{#useAndroidMavenGradlePlugin}}
+        classpath 'com.github.dcendents:android-maven-plugin:1.2'
+        {{/useAndroidMavenGradlePlugin}}
+    }
+}
+
+allprojects {
+    repositories {
+        jcenter()
+    }
+}
+
+
+apply plugin: 'com.android.library'
+{{#useAndroidMavenGradlePlugin}}
+apply plugin: 'com.github.dcendents.android-maven'
+{{/useAndroidMavenGradlePlugin}}
+
+android {
+    compileSdkVersion 22
+    buildToolsVersion '22.0.0'
+    defaultConfig {
+        minSdkVersion 14
+        targetSdkVersion 22
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_7
+        targetCompatibility JavaVersion.VERSION_1_7
+    }
+
+    // Rename the aar correctly
+    libraryVariants.all { variant ->
+        variant.outputs.each { output ->
+            def outputFile = output.outputFile
+            if (outputFile != null && outputFile.name.endsWith('.aar')) {
+                def fileName = "${project.name}-${variant.baseName}-${version}.aar"
+                output.outputFile = new File(outputFile.parent, fileName)
+            }
+        }
+    }
+}
+
+
+ext {
+    swagger_annotations_version = "1.5.0"
+    gson_version = "2.3.1"
+    httpclient_version = "4.3.3"
+    junit_version = "4.8.1"
+}
+
+dependencies {
+    compile "io.swagger:swagger-annotations:$swagger_annotations_version"
+    compile "com.google.code.gson:gson:$gson_version"
+    compile "org.apache.httpcomponents:httpcore:$httpclient_version"
+    compile "org.apache.httpcomponents:httpmime:$httpclient_version"
+    testCompile "junit:junit:$junit_version"
```

Comment:
Hi
Sorry I missed this.

compile `compile "com.mcxiaoke.volley:library:${volley_version}@aar"` should be added to build.mustache, yes. Also     `volley_version = "1.0.19"` in the `ext` section.

I think compileSdkVersion might be updated to `23` because it is the lasestes (Android 6), that's why I used it in the samples.

Should I correct these and send a new PR?